power_grid_model_io_native_c/power_grid_model_io_native/include/power_grid_model_io_native/pgm_vnf_converter/vnf_parser.hpp

```diff
@@ -0,0 +1,100 @@
+// SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#pragma once
+#ifndef POWER_GRID_MODEL_IO_NATIVE_C_VNF_PARSER_HPP
+#define POWER_GRID_MODEL_IO_NATIVE_C_VNF_PARSER_HPP
```

Comment:
cfr. to conventions used in PGM: include guards other than `#pragma once` aren't really necessary unless it's exposed in the C API. The reason is that most compilers support `#pragma once` and the `ifndef` include guards are only for those that do not support it when another build host/compiler is used when using the C API than for building it.

it's not worth applying a separate fix but just for future reference.